src/gui/map/map_editor.cpp

```diff
@@ -2068,7 +2068,7 @@ void MapEditorController::createTagSelector()
 	auto tag_widget = new TagSelectWidget(map, main_view, this);
 	tag_select_dock_widget = new EditorDockWidget(tr("Tag Selector"), tag_select_window_act, this, window);
 	tag_select_dock_widget->setWidget(tag_widget);
-	tag_select_dock_widget->setObjectName(QString::fromLatin1("Tag selection dock widget"));
+	tag_select_dock_widget->setObjectName(QString::fromLatin1("Tag selector dock widget"));
```

Comment:
This is not relevant for the user interface, only needed for properly storing/restoring window state.